heft-plugins/heft-sass-plugin/src/utilities/Async.ts

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import { ScopedLogger } from '@rushstack/heft';
+import type { IScopedLogger } from '@rushstack/heft';
 
 export class Async {
```

Comment:
Seems like this should be a standard utility function somewhere. Maybe in a `@rushstack/heft-plugin-utilities` package? I know @octogonz doesn't like the name "utilities," so I'm down with another name.

Reply:
I could see value there, but it's likely this (as well as the other use of this method in the Heft PR) are going to be removed when I update the way we do watch mode (since the plugin won't need to provide it's own watcher)